src/pages/images/ImageSelector.tsx

```diff
@@ -264,7 +264,7 @@ const ImageSelector: FC<Props> = ({ onClose, onSelect }) => {
                 setRelease(v.target.value);
               }}
               options={getOptionList(
-                (item) => item.release,
+                (item) => item.release_title,
```

Comment:
By and large this should work, because we already have the shortname in the alias column, but that's not always the case, e.g.:

![image](https://github.com/canonical/lxd-ui/assets/56583786/c4184ef3-119c-4bd7-ac57-d93b33f21afd)

I think the original request from @lathiat was to show both. In this case the shortname "jammy" is not shown, could that be a problem?